doc/user_guide/tools.rst

```diff
@@ -928,8 +927,61 @@ method that operates *in place*.
 Rebinning
 ^^^^^^^^^
 
-The :py:meth:`~.signal.BaseSignal.rebin` method rebins data in place down to a size
-determined by the user.
+Rebinning of the spectral data will be carried out by two methods.
+The fast method will run if the new_shape is a divisor of the original shape.
+Otherwise a slower linear interpolation method will be applied. Both methods are incorporated into the :py:meth:`~.signals.eds.rebin` function.
```

Comment:
Not groundbreaking, but it should probably not point to specifically `eds.rebin` method.